onnxruntime/test/optimizer/qdq_test_utils.h

```diff
@@ -134,67 +134,6 @@ GetQDQTestCaseFn BuildQDQInstanceNormTestCase(const std::vector<int64_t>& input_
   };
 }
 
-// Creates the following graph if axes is an input (newer opsets):
-//                                _______________________
-//    input (f32) -> Q -> DQ ->  |                       | -> Q -> DQ -> output (f32)
-// axes (int32, initializer) ->  |       Reduce___       |
-//                               |_______________________|
-//
-// Creates the following graph if axes is an attribute (older opsets):
-//                                _______________________
-//    input (f32) -> Q -> DQ ->  |                       | -> Q -> DQ -> output (f32)
-//                               |       Reduce___       |
-//                               |_______________________|
-//
-// Currently used to test QNN EP.
-template <typename QuantType>
-GetQDQTestCaseFn BuildQDQReduceOpTestCase(const std::string& reduce_op_type, const std::vector<int64_t>& input_shape,
```

Comment:
I originally introduced this test function for QNN. No one else is using it, and I need to be able to modify it freely, so moved to qnn testing directory.